src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const ResultObject = [];
```

Comment:
```suggestion
  const stateClones = [];
```
camelCase for variable names
use meaningful names (it's not an object - it's array with state clones)